schedule/yast/select_modules_and_patterns+registration/select_modules_and_patterns+registration.yaml

```diff
@@ -43,6 +43,6 @@ schedule:
   - shutdown/grub_set_bootargs
   - console/validate_installed_packages
   - console/validate_installed_patterns
-  - console/suseconnect_scc
+  - installation/addon_products_via_SCC_yast2_libyui
```

Comment:
we should find a name that represent a test case in the installed system (so without path `installation/` and without `libyui` name on it, a name more related with functional testing than the technology used under the hood.